djstripe/models.py

```diff
@@ -158,6 +159,16 @@ class Customer(StripeCustomer):
     )
     date_purged = DateTimeField(null=True, editable=False)
 
+    coupon = ForeignKey(Coupon, null=True, on_delete=SET_NULL)
+    coupon_start = StripeDateTimeField(
```

Comment:
You're right... putting these here is super weird. Let's just blame it on the person who designed the coupon infrastructure.